lua/nightfox/group/modules/navic.lua

```diff
@@ -0,0 +1,39 @@
+-- https://github.com/SmiteshP/nvim-navic
+
+local M = {}
+
+function M.get(spec, config, opts)
+  -- stylua: ignore
+  return {
+    NavicText               = { fg = spec.fg1 },
+    NavicSeparator          = { fg = spec.bg4 },
+
+    NavicIconsFile          = { link = "Directory" },
+    NavicIconsModule        = { link = "TSNamespace" },
```

Comment:
`TS*` highlight groups are the old deprecated format. You should use the new `@` format. See the [treesitter](https://github.com/EdenEast/nightfox.nvim/blob/main/lua/nightfox/group/modules/treesitter.lua) module file for more examples.

As an example `TSNamespace` -> `@namespace`.

Reply:
Updated them, thanks!